pset-iterators.js

```diff
@@ -171,6 +206,22 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+
+const shortTermPresidents = presidents.filter(lessThan4 => {
+
+  let fuckDonaldTrump = presidents.pop()
+
+   let yearLeft = lessThan4.left_office.slice(0,4)
+   let yearStart = lessThan4.took_office.slice(0,4)
+
+   let numberOfYears = yearLeft - yearStart
+   if (numberOfYears < 4) {
```

Comment:
🛠 Read the comment below and update your code accordingly

@Defy1x fix your indentation

Reply:
Done